packages/passport-auth/lib/Google.js

```diff
@@ -10,7 +10,7 @@ class GoogleAuthStrategy extends PassportAuthStrategy {
       {
         sessionIdField: 'googleSession',
         keystoneSessionIdField: 'keystoneGoogleSessionId',
-        scope: ['profile'],
+        scope: ['openid profile', 'openid email'],
```

Comment:
I don't remember but is adding more scopes gets you a user consent screen on google side?
users should be able to add more scope in config if needed.

Reply:
Yeah, it probably will. I think its ok to have this as a default, but allow the app developer to overwrite it if they want less scope. In my experience, most apps want email anyway, so this makes the default work as expected for them.